### PR TITLE
Have stop_grace_period also set StopTimeout on create

### PR DESCRIPTION
**Pull request by andyneff**
Feature to handle #5310

### CODE REVIEW

File: compose/service.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 from docker.errors import NotFound
 from docker.types import LogConfig
 from docker.utils import version_lt
+from docker.utils import version_gte
```
Comment (shin-) [inline compose/service.py]: Make sure those are ordered, right now you're failing the pre-commit checks.

File: compose/service.py [hunk 2]
```diff
@@ -759,6 +760,11 @@ def _get_container_create_options(
             container_options['hostname'] = parts[0]
             container_options['domainname'] = parts[2]
 
+        if (version_gte(self.client.api_version, '1.25') and
+                'stop_grace_period' in self.options):
+            container_options['stop_timeout'] = parse_seconds_float(
```
Comment (shin-) [inline compose/service.py]: use `self.stop_timeout(None)` instead

**Comment by andyneff** [comment]
@shin- Changes made. Let me know if there's anything else I need to do.

**Comment by shin-** [comment]
LGTM, thank you!

**Pull request merged**